.env.example

```diff
@@ -1,4 +1,4 @@
-ADMIN_EMAIL="kinetic@kin.org"
+ADMIN_USERNAME="alice"
```

Comment:
@RamirezAlex make sure to update your `.env` and do a `yarn prisma migrate reset && yarn prisma db push`